e2e/app/page/google-login.ts

```diff
@@ -84,22 +86,16 @@ export default class GoogleLoginPage extends BasePage {
    * Click Next button to submit login credential.
    */
   async submit() : Promise<void> {
-    const button = await this.page.waitForXPath(SELECTOR.submitButton, {visible: true});
-    await this.clickAndWait(button);
+    const button = await this._page.waitForXPath(SELECTOR.submitButton, {visible: true});
+    await BaseElement.asBaseElement(this._page, button).clickAndWait();
   }
 
   /**
    * Open All-of-Us Google login page.
    */
   async load(): Promise<void> {
     const url = config.uiBaseUrl + config.loginUrlPath;
-    try {
-      await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 0});
-    } catch (err) {
-      console.error('Google login page not found. ' + err);
-      await takeScreenshot(this.page, 'GoogleLoginPageNotFound');
-      throw err;
-    }
+    await this._page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 120000});
```

Comment:
please define these two string constants with better names. I realize they're external.

That is a huge default timeout. I'd rather default to reasonable ones and only give more slack if we can't avoid it.